experiments/scripts/demoChecker.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Default values
+INPUT_DIR_COUNT=0
+INPUT_DIRS=()
+OUTPUT_DIR=$HOME/.demoChecker
```

Comment:
Your default here is to create a hidden file in `$HOME`, I don't think it is a good idea because users would have a hard time to find it afterwards. I think a better default would be to create a "demoChecker" in the current directory:

```
OUTPUT_DIR=demoChecker
```